contrib/templates/experimental/scala-job/template/build.sbt.tmpl

```diff
@@ -0,0 +1,22 @@
+scalaVersion := "{{template `scala_version` .}}"
+
+name := "{{.project_name}}"
+organization := "com.examples"
+version := "{{template `version` .}}"
+
+libraryDependencies += "com.databricks" % "databricks-connect" % "{{template `dbr_version` .}}.+"
+libraryDependencies += "org.slf4j" % "slf4j-simple" % "2.0.16"
+
+assembly / assemblyOption ~= { _.withIncludeScala(false) }
+assembly / assemblyExcludedJars := {
+  val cp = (assembly / fullClasspath).value
+  cp filter { _.data.getName.matches("scala-.*") } // remove Scala libraries
+}
+
+assemblyMergeStrategy := {
+  case _ => MergeStrategy.preferProject
+}
+
+// to run with new jvm options, a fork is required otherwise it uses same options as sbt process
+run / fork := true
+run / javaOptions += "--add-opens=java.base/java.nio=ALL-UNNAMED"
```

Comment:
This folder should also have a .gitignore file. Basis for that: https://github.com/databricks/cli/blob/main/libs/template/templates/default-python/template/%7B%7B.project_name%7D%7D/.gitignore